### DISCUSSION TITLE
Internal Server Error Page

### DISCUSSION THREAD

**Discussion by rhonalindiefolks** [pretix/pretix#3503] [Q&A] [answered]
Hello Guys. 

I am doing a plugins for override the error page "internal error page" and " Page no found", There is a way to change the default page or change the html. 

Thank you!

**Comment by raphaelm** [comment, marked as answer]
No, there is no intended way to do this. You can of course override/replace the corresponding templates.

**Reply by rhonalindiefolks** [reply]
Thank you
